aw-server/src/endpoints/key_value.rs

```diff
@@ -0,0 +1,37 @@
+use std::sync::MutexGuard;
+use rocket_contrib::json::Json;
+use rocket::http::Status;
+use rocket::State;
+use crate::endpoints::ServerState;
+
+use aw_datastore::{Datastore, DatastoreError};
+
+#[post("/<key>", data="<message>")]
+pub fn value_new(state: State<ServerState>, key: String, message: Json<String>)
+    -> Result<Status, Status> {
+
+    let data = message.into_inner();
+    let datastore: MutexGuard<'_, Datastore> = endpoints_get_lock!(state.datastore);
+    let result = datastore.create_value(&key, &data);
+    return match result {
+        Ok(_) => Ok(Status::Created),
```

Comment:
Should it be the same status no matter whether if it's created or replaced?
Probably not, but maybe not very important to have that distinction for now either. Maybe just add a "TODO" comment for now if it's a significant amount of work.

Reply:
I think it should be the same for created and updated. I don't see a reason for why you'd want to know the difference.